.github/scripts/classify-wework-desktop-e2e.sh

```
@@ -194,10 +269,36 @@ classify_wework_path() {
       return
       ;;
 
+    # Managed Worktree availability, routing, task projection, and settings
+    # must keep both the local launch path and the real cloud lifecycle green.
+    wework/src/api/executorAccess* | \
+      wework/src/api/hybrid/hybridServices* | \
+      wework/src/api/local/localServices* | \
+      wework/src/api/runtimeWork* | \
```

Shellcheck (0.11.0)

[warning] This pattern always overrides a later one on line 346.

(SC2221)